package.json

```diff
@@ -229,5 +229,8 @@
       "npx gdc-fix-copyright",
       "git add"
     ]
+  },
+  "engines": {
+    "node": "^8.9.4"
```

Comment:
Please unify the node version
[SDK document](https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html): Node ^8.10.0 or >=9.10.0
[Ticket](https://jira.intgdc.com/browse/FET-389): 8.12.0

Reply:
It's `^8.10.0` now